### PR TITLE
Fix and normalize JSON code and JSON tagging for Sensu Enteprise

### PR DESCRIPTION
**Pull request by rgeniesse**
## Description
Related to #663.

Fix JSON code, use `shell` highlighting as we do elsewhere on the site and fix some JSON end tags.

## Motivation and Context
Consistency and providing users with valid JSON to start with.

## How Has This Been Tested?
Locally with `yarn` and `jq`. Overall with JSON validator.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New doc/guide
- [X] Fixing errata
- [ ] Update (Add missing or refresh existing content)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] All tests have passed.

### CODE REVIEW

File: content/sensu-enterprise/2.6/guides/using-snow.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ In order for the SNOW configuration to shine, we need to set attributes for SNOW
 
 {{< highlight json >}}{
  "client": {
-   ...
+   "...": "...",
```
Comment (apaskulin) [inline content/sensu-enterprise/2.6/guides/using-snow.md:66]: Brilliant! More manageable samples, but still valid JSON

File: content/sensu-enterprise/2.8/integrations/influxdb.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ description    | Configurable custom tags (key/value pairs) to add to every Infl
 required       | false
 type           | Hash
 default        | {{< highlight shell >}}{}{{</ highlight >}}
-example        | {{< highlight json >}}
+example        | {{< highlight shell >}}
```
Comment (apaskulin) [inline content/sensu-enterprise/2.8/integrations/influxdb.md]: Curious about this: Why change to shell?

Comment (rgeniesse) [reply content/sensu-enterprise/2.8/integrations/influxdb.md]: It conforms what I have seen in other examples within tables. A few examples before I started this effort:

https://github.com/sensu/sensu-docs/blame/master/content/sensu-core/1.4/reference/checks.md#L673-L687
https://github.com/sensu/sensu-docs/blame/master/content/sensu-core/1.4/reference/checks.md#L795-L801

I think though now that you have brought it up, it requires further discussion on when we use JSON take vs shell. IMO, it should be:

- JSON tags for full, valid JSON
- shell tags for JSON objects that can go in as Sensu attributes

If we made them all JSON tags, we'd have to add `{}` to all the JSON objects that can be Sensu attributes.

Comment (apaskulin) [reply content/sensu-enterprise/2.8/integrations/influxdb.md]: That makes sense to me! Thanks for elaborating.

**Review by apaskulin — APPROVED** [review]
💫

**Pull request merged**